tslint.json

```diff
@@ -68,12 +68,14 @@
     "variable-name": false,
     "whitespace": [true, "check-branch", "check-decl", "check-operator", "check-separator", "check-type"],
     "no-output-on-prefix": true,
-    "directive-selector": [true, "attribute", "sfeir", "camelCase"],
+    "directive-selector": [true, "attribute", "sfeir", "camelCase", "kebab-case"],
     "component-selector": [true, "element", "sfeir", "kebab-case"],
     "use-input-property-decorator": true,
     "use-output-property-decorator": true,
     "use-host-property-decorator": true,
-    "no-input-rename": true,
+    "no-input-rename": {
+      "severity": "warn"
```

Comment:
same thing as above: I think it's a bad practice. Fixing the use cases instead of changing the rules would be better.